<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="IBM-965GDN3" failures="0" tests="2" name="appium.MobileTesting.Logintest" time="152.737" errors="0" timestamp="02 Jan 2024 14:38:32 GMT" skipped="0">
  <testcase classname="appium.MobileTesting.Logintest" name="testAppium2" time="0.002"/>
  <testcase classname="appium.MobileTesting.Logintest" name="testAppium1" time="152.735"/>
=======
<testsuite hostname="IBM-965GDN3" failures="0" tests="1" name="appium.MobileTesting.Logintest" time="184.537" errors="1" timestamp="11 Jan 2024 14:46:34 GMT" skipped="0">
  <testcase classname="appium.MobileTesting.Logintest" name="testAppium1" time="184.537">
    <error message="Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at &#039;C:\Users\0047HE744\Desktop\Personal\MyLearning\AppiumTesting\MobileTesting/consumer-app.apk&#039; does not exist or is not accessible 
Host info: host: &#039;IBM-965GDN3&#039;, ip: &#039;192.168.0.190&#039;
Build info: version: &#039;4.16.1&#039;, revision: &#039;9b4c83354e&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.5&#039;
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{appium:autoGrantPermissions=true, appium:deviceName=emulatortest2, platformName=ANDROID, appium:automationName=UIAutomator2, appium:app=C:\Users\0047HE744\Desktop\Personal\MyLearning\AppiumTesting\MobileTesting/consumer-app.apk}], desiredCapabilities=Capabilities {appium:app: C:\Users\0047HE744\Desktop\..., appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: emulatortest2, platformName: ANDROID}}]
Capabilities {appium:app: C:\Users\0047HE744\Desktop\..., appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: emulatortest2, platformName: ANDROID}" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\Users\0047HE744\Desktop\Personal\MyLearning\AppiumTesting\MobileTesting/consumer-app.apk' does not exist or is not accessible 
Host info: host: 'IBM-965GDN3', ip: '192.168.0.190'
Build info: version: '4.16.1', revision: '9b4c83354e'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.5'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{appium:autoGrantPermissions=true, appium:deviceName=emulatortest2, platformName=ANDROID, appium:automationName=UIAutomator2, appium:app=C:\Users\0047HE744\Desktop\Personal\MyLearning\AppiumTesting\MobileTesting/consumer-app.apk}], desiredCapabilities=Capabilities {appium:app: C:\Users\0047HE744\Desktop\..., appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: emulatortest2, platformName: ANDROID}}]
Capabilities {appium:app: C:\Users\0047HE744\Desktop\..., appium:autoGrantPermissions: true, appium:automationName: UIAutomator2, appium:deviceName: emulatortest2, platformName: ANDROID}
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:140)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:133)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:102)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:177)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:238)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:523)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:268)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:164)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:90)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:102)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)
at appium.MobileTesting.AppiumSetup.initiateDriver(AppiumSetup.java:118)
at appium.MobileTesting.Logintest.testAppium1(Logintest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testAppium1 -->
>>>>>>> ea2465ad2fffcdafc79d827c8483d7f4f76fc277
</testsuite> <!-- appium.MobileTesting.Logintest -->
